CMakeLists.txt

```diff
@@ -184,6 +185,13 @@ else()
   pkg_check_modules(PORTAUDIO REQUIRED portaudio-2.0)
 endif()
 
+# include zitaconvolver
+if (USE_INTERNAL_ZITACONVOLVER)
+  set(ZITACONVOLVER_INCLUDE_DIRS ${CMAKE_SOURCE_DIR}/submodules/ZitaConvolver/source)
+else()
```

Comment:
It would be more clear to set `ZITACONVOLVER_INCLUDE_DIRS` and `ZITACONVOLVER_LIBRARIES` to empty values here.